account_bank_statement_import_auto_reconcile/models/account_bank_statement_import_reapply_rules.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, fields, models
+from odoo.exceptions import Warning as UserError
+
+
+class AccountBankStatementImportReapplyRules(models.TransientModel):
```

Comment:
What is the purpose of reapplying rules and when would that be used? Again I miss some documentation/explanation.

Reply:
Well, the rules are applied on the import of a statement and the lines are reconciled then. Apparently this goes through the rules and reconciles again. 

I am not sure about the author's intentions though, besides what I can see on code. 

cc @hbrunn

Reply:
when you change the matching configuration, you can use this to reapply your changed rules on an existing bank statement to 1) see if it matches stuff it didn't match before 2) not need to do the reconciliations manually. Same thing if you have new move lines since the import of the statement (might come from some other system).